cluster/plan.go

```diff
@@ -558,6 +560,26 @@ func (c *Cluster) BuildKubeletProcess(host *hosts.Host, serviceOptions v3.Kubern
 			fmt.Sprintf("%s:c:/host/run", path.Join(host.PrefixPath, "/run")),
 		}...)
 	} else {
+		parsedVersion, err := getClusterVersion(c.Version)
+		if err != nil {
+			logrus.Debugf("Error while parsing cluster version: %s", err)
```

Comment:
Not a fan of "shutting up" this error message, but I see it's an established pattern in this file.
So nothing to address, just sharing my thoughts here for completeness of the review.